addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1813,6 +1813,9 @@ const ColorpickerUserValueWidget = SelectUserValueWidget.extend({
         if (wysiwyg) {
             options.document = this.$target[0].ownerDocument;
             options.getTemplate = wysiwyg.getColorpickerTemplate.bind(wysiwyg);
+            // Load the getEditableCustomColors from the background props.
+            // This function is a common option for both background and text colorpalettes
```

Comment:
Nitpicking: max 80 chars per line for comments.
But I would remove this comment entirely:
1. It explains a one line code by stating exactly what it does: assigning something. Not useful :wink: 
2. The second line is actually not precise: we have colorpickers for more than background and text (e.g. shape sub-colors) or even theme colors. It's just a feature that we want for all colorpickers whatever their purpose is, no need to state it, it's obvious from the context.